ntoskrnl/se/token.c

```diff
@@ -1086,6 +1086,17 @@ SepDuplicateToken(
         goto Quit;
     }
 
+    /* Fill in token debug information */
+#if DBG
+    RtlCopyMemory(AccessToken->ProcessName,
+                  PsGetCurrentProcess()->ImageFileName,
```

Comment:
Regarding the bug mentioned on Mattermost, there exists the case, where tokens start to be first used in the kernel, before any process (including the "initial" kernel process) get created; therefore in that case, `PsGetCurrentProcess()` would return `NULL` and you're in trouble.